meeshkan/serve/mock/faker/schema_faker.py

```diff
@@ -0,0 +1,278 @@
+########################
```

Comment:
As I understand it, none of the calls to `self` in the class are stateful. I may be wrong, though!
If it is not stateful, I would recommend removing the class, renaming `self` to `deps: FakerDependencies`, and then use a `FakerDependencies` `dataclass` to do dependency injection. If there is no stateful behavior, it's a simple refactor and it communicates clearly that the faker is not supposed to retain state.

Reply:
I made it stateless with a dataclass passed through.